packages/theme/layouts/error.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="error-page">
+    <h1 v-if="error.statusCode === 404">
+      {{ $t('Page not found') }}
+    </h1>
+    <h1 v-else>
+      {{ $t('An error occurred') }}
+    </h1>
+    <SfButton :link="localePath('/')">
+      {{ $t('Back to Home page') }}
+    </SfButton>
+  </div>
+</template>
+
+<script>
+import { SfButton } from '@storefront-ui/vue';
+
+export default {
```

Comment:
```suggestion
export default defineComponent({
```
For better TypeScript support with Volar